### PR TITLE
[Benchmarks] Add the WTX project

### PR DESCRIPTION
**Pull request by c410-f3r**
~~WIP. I will add the remaining `bench_loading_associations_sequentially` benchmark group as soon as 1.75 hit stable.~~

Edit: NVM, it is already finished.

### CODE REVIEW

File: .github/workflows/metrics.yml [hunk 1]
```diff
@@ -75,6 +75,10 @@ jobs:
         if: matrix.backend == 'mysql'
         run: cargo +stable bench --manifest-path diesel_bench/Cargo.toml --no-default-features --features "mysql sqlx-bench sqlx/mysql tokio rustorm rustorm/with-mysql rustorm_dao rust_mysql futures sea-orm sea-orm/sqlx-mysql criterion/async_tokio quaint quaint/mysql quaint/serde-support serde diesel-async diesel-async/mysql"
 
+      - name: Run Benchmarks (Wtx)
+        if: matrix.backend == 'postgres'
+        run: cargo +stable bench --manifest-path diesel_bench/Cargo.toml --no-default-features --features "postgres tokio/rt-multi-thread wtx"
```
Comment (weiznich) [inline .github/workflows/metrics.yml:80]: Is there any reason not to add this to the postgres job above?

**Review by weiznich — COMMENTED** [review]
Thanks for opening this PR. This looks good to be merged, beside the minor thing about integration into the CI setup.

File: .github/workflows/metrics.yml (hunk 1 above)
Comment (c410-f3r) [reply .github/workflows/metrics.yml]: Ops, didn't notice. Moved

**Review by weiznich — APPROVED** [review]

**Comment by c410-f3r** [comment]
```
 assertion `left == right` failed
  left: [("Sean says hi", Some("updated")), ("Tess says hi", Some("updated")), ("Ruby says hi", None)]
 right: [("Tess says hi", Some("updated")), ("Sean says hi", Some("updated")), ("Ruby says hi", None)]
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Unrelated

**Pull request merged**